src/asim/configs/resident/summarize_preprocessor.csv

```diff
@@ -0,0 +1,434 @@
+,_hov2_l_tolldist,trip_odt_skims['HOV2_L_TOLLDIST']
+,_hov2_m_tolldist,trip_odt_skims['HOV2_M_TOLLDIST']
+,_hov2_h_tolldist,trip_odt_skims['HOV2_H_TOLLDIST']
+,_hov3_l_tolldist,trip_odt_skims['HOV3_L_TOLLDIST']
+,_hov3_m_tolldist,trip_odt_skims['HOV3_M_TOLLDIST']
+,_hov3_h_tolldist,trip_odt_skims['HOV3_H_TOLLDIST']
+,_hov2_l_hovdist,trip_odt_skims['HOV2_L_HOVDIST']
+,_hov2_m_hovdist,trip_odt_skims['HOV2_M_HOVDIST']
+,_hov2_h_hovdist,trip_odt_skims['HOV2_H_HOVDIST']
+,_hov3_l_hovdist,trip_odt_skims['HOV3_L_HOVDIST']
+,_hov3_m_hovdist,trip_odt_skims['HOV3_M_HOVDIST']
+,_hov3_h_hovdist,trip_odt_skims['HOV3_H_HOVDIST']
+
+# Transit auto access/egress distance (miles x 100)
+#####,_drv_com_wlk_ddist,trip_odt_skims['DRV_COM_WLK_DDIST']/100
+#####,_drv_exp_wlk_ddist,trip_odt_skims['DRV_EXP_WLK_DDIST']/100
+#####,_drv_hvy_wlk_ddist,trip_odt_skims['DRV_HVY_WLK_DDIST']/100
+#####,_drv_loc_wlk_ddist,trip_odt_skims['DRV_LOC_WLK_DDIST']/100
+#####,_drv_lrf_wlk_ddist,trip_odt_skims['DRV_LRF_WLK_DDIST']/100
+
+# Bike distance
+#####,_bike_dist,trip_od_skims['DISTBIKE'] CHECK
+,_bike_dist,trip_od_skims['DIST']
+
+# Walk distance
+#####,_walk_dist,trip_od_skims['DISTWALK'] CHECK
+,_walk_dist,trip_od_skims['DIST']
+
+## Times
+# Auto time
+##### loses toll times
+,_sov_tr_l_time,trip_odt_skims['SOV_TR_L_TIME']
+,_sov_tr_m_time,trip_odt_skims['SOV_TR_M_TIME']
+,_sov_tr_h_time,trip_odt_skims['SOV_TR_H_TIME']
+,_sov_nt_l_time,trip_odt_skims['SOV_NT_L_TIME']
+,_sov_nt_m_time,trip_odt_skims['SOV_NT_M_TIME']
+,_sov_nt_h_time,trip_odt_skims['SOV_NT_H_TIME']
+,_hov2_l_time,trip_odt_skims['HOV2_L_TIME']
+,_hov2_m_time,trip_odt_skims['HOV2_M_TIME']
+,_hov2_h_time,trip_odt_skims['HOV2_H_TIME']
+,_hov3_l_time,trip_odt_skims['HOV3_L_TIME']
+,_hov3_m_time,trip_odt_skims['HOV3_M_TIME']
+,_hov3_h_time,trip_odt_skims['HOV3_H_TIME']
+
+
+# Transit vehicle time
+,_knrin_loc_wlk_totivt,trip_odt_skims['KNRIN_LOC_TOTALIVTT']
+,_knrin_mix_wlk_totivt,trip_odt_skims['KNRIN_MIX_TOTALIVTT']
+,_knrin_prm_wlk_totivt,trip_odt_skims['KNRIN_PRM_TOTALIVTT']
+,_pnrin_loc_wlk_totivt,trip_odt_skims['PNRIN_LOC_TOTALIVTT']
+,_pnrin_mix_wlk_totivt,trip_odt_skims['PNRIN_MIX_TOTALIVTT']
+,_pnrin_prm_wlk_totivt,trip_odt_skims['PNRIN_PRM_TOTALIVTT']
+,_tncin_loc_wlk_totivt,trip_odt_skims['TNCIN_LOC_TOTALIVTT']
+,_tncin_mix_wlk_totivt,trip_odt_skims['TNCIN_MIX_TOTALIVTT']
+,_tncin_prm_wlk_totivt,trip_odt_skims['TNCIN_PRM_TOTALIVTT']
+,_knrout_loc_wlk_totivt,trip_odt_skims['KNROUT_LOC_TOTALIVTT']
+,_knrout_mix_wlk_totivt,trip_odt_skims['KNROUT_MIX_TOTALIVTT']
+,_knrout_prm_wlk_totivt,trip_odt_skims['KNROUT_PRM_TOTALIVTT']
+,_pnrout_loc_wlk_totivt,trip_odt_skims['PNROUT_LOC_TOTALIVTT']
+,_pnrout_mix_wlk_totivt,trip_odt_skims['PNROUT_MIX_TOTALIVTT']
+,_pnrout_prm_wlk_totivt,trip_odt_skims['PNROUT_PRM_TOTALIVTT']
+,_tncout_loc_wlk_totivt,trip_odt_skims['TNCOUT_LOC_TOTALIVTT']
+,_tncout_mix_wlk_totivt,trip_odt_skims['TNCOUT_MIX_TOTALIVTT']
+,_tncout_prm_wlk_totivt,trip_odt_skims['TNCOUT_PRM_TOTALIVTT']
```

Comment:
There is no 'TNCOUT_LOC_TOTALIVTT', 'TNCOUT_MIX_TOTALIVTT', 'TNCOUT_PRM_TOTALIVTT' in transit skims